src/module/display-home.js

```diff
@@ -0,0 +1,83 @@
+import showPopupComment from './show-popup-comment.js';
+import addLike from './add-like.js';
+import count from './count-items.js';
+import showlikes from './display-likes.js';
+// import displayComments from './display-comments.js';
```

Comment:
As requested by the previous reviewer, kindly remove the commented code from your code base in order to make it more professional and clean. :wink: